### PR TITLE
Distify

### PR DESCRIPTION
**Pull request by zdykstra**
This PR removes all of the legacy Jenkins / Debian packaging files. Instead, it's deployed as a minimal application to our darkpan. See the corresponding changes to the scn.nagios-ehf role.

This can be installed manually via `cpanm SCN::App::NagiosEHF --mirror https://darkpan.servercentral.com/combined/`

### CODE REVIEW

File: MANIFEST.SKIP [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
+#!start included /home/jberger/perl5/perlbrew/perls/perl-5.22.1/lib/5.22.1/ExtUtils/MANIFEST.SKIP
```
Comment (andrewmeyer) [inline MANIFEST.SKIP:2]: should this be generalized?

**Review by andrewmeyer — COMMENTED** [review]
MANIFEST.SKIP.bak and the jberger lines seem confusing to me.  Is it normal for our distify process to generate a `.bak` file and use jberger's home dir?

File: MANIFEST.SKIP (hunk 1 above)
Comment (zdykstra) [reply MANIFEST.SKIP:2]: That's an auto-generated comment by helper tools. The path in there can be ignored, it's informational only.

Comment (jberger) [reply MANIFEST.SKIP:2]: You've discovered some of the really dumb things about the MANIFEST.SKIP system. The original skip file is generated and it imprints the file location of the original that it included as a comment. Yes that's just a comment you could remove it but meh. The .bak file happens when you generate it twice and it tries very hard not to overwrite, at least once.

Comment (jberger) [reply MANIFEST.SKIP:2]: None of it should matter and for internal tooling I'd just leave it alone.

**Review by andrewmeyer — APPROVED** [review]

**Pull request merged**